aiqc/tests/__init__.py

```diff
@@ -1125,6 +1198,8 @@ def make_test_queue(name:str, repeat_count:int=1, fold_count:int=None):
 		queue = make_test_queue_keras_multiclass(repeat_count, fold_count)
 	elif (name == 'keras_binary'):
 		queue = make_test_queue_keras_binary(repeat_count, fold_count)
+	elif (name == 'keras_text_binary'):
```

Comment:
I'll add a line for this to the PR template tests

Reply:
thanks